.github/workflows/python.yml

```diff
@@ -40,10 +40,34 @@ jobs:
     timeout-minutes: 120
     steps:
       - uses: actions/checkout@v1
+      - name: Create the kind config
+        run: |
+          configFile=/home/runner/work/submarine/submarine/kind-config-kind.yaml
```

Comment:
`/home/runner/work/submarine/submarine/kind-config-kind.yaml` is this path fixed?

Reply:
If we set config: ./kind-config-kind.yaml (line: 70), KinD will set the path to a config file under "/home/runner/work/submarine/submarine/".
I can replace the configFile variable to "./kind-config-kind.yaml" (just replace it with the relative path)